pkg/api/resolver_model_edit.go

```diff
@@ -58,9 +58,27 @@ func (r *editResolver) Expires(ctx context.Context, obj *models.Edit) (*time.Tim
 		startTime = obj.UpdatedAt.Time
 	}
 
+	fac := r.getRepoFactory(ctx)
+	qb := fac.Edit()
+	editVotes, err := qb.GetVotes(obj.ID)
```

Comment:
This seems worrisome that I have to go back to the DB to get the votes in order to calculate the expires time for each entry. It's the exact same code as the GetVotes method further down in this file.
Is there any caching happening so that maybe it won't have to hit the database twice for the same data for every edit?